tree/ntuple/v7/inc/ROOT/RMiniFile.hxx

```diff
@@ -17,6 +17,7 @@
 #define ROOT7_RMiniFile
 
 #include <ROOT/RError.hxx>
+#include <ROOT/RNTupleAnchor.hxx>
```

Comment:
this makes this slightly less minimal but maybe it's simple enough for reader to just read it?

Reply:
Not sure I understand that. What do you mean by "reader just read it"?

Reply:
I see. I think the complexity is not in the `RNTupleAnchor`.